testenv/utils/docker/docker.go

```diff
@@ -82,6 +83,27 @@ func (e *DockerHelper) Services(ctx context.Context) (envtypes.Services, error)
 	return serviceCollection.AsServices(), nil
 }
 
+func (e *DockerHelper) WaitForDockerReady(ctx context.Context) error {
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, 5*time.Minute)
```

Comment:
its okay to redeclare ctx here (reducing to 2 minutes should also be okay)
```suggestion
	ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
```

Reply:
2 minutes is not enough for GCP :S